dialogue-serde/src/test/java/com/palantir/conjure/java/dialogue/serde/DefaultClientsTest.java

```diff
@@ -60,4 +70,19 @@ public void testCall() throws ExecutionException, InterruptedException {
         assertThat(result).isDone();
         assertThat(result.get()).isEqualTo("value");
     }
+
+    @Test
+    public void testAddsAcceptHeader() throws ExecutionException, InterruptedException {
+        Request request = Request.builder().build();
+        when(deserializer.deserialize(eq(response))).thenReturn("value");
+        String expectedAccept = "application/json";
+        when(deserializer.accepts()).thenReturn(Optional.of(expectedAccept));
+        when(channel.execute(eq(endpoint), any())).thenReturn(Futures.immediateFuture(response));
+        ListenableFuture<String> result = DefaultClients.INSTANCE.call(channel, endpoint, request, deserializer);
+        assertThat(result).isDone();
+        assertThat(result.get()).isEqualTo("value");
+        verify(channel).execute(eq(endpoint), requestCaptor.capture());
+        assertThat(requestCaptor.getValue().headerParams().get(HttpHeaders.ACCEPT))
+                .containsExactly(expectedAccept);
```

Comment:
nit: I find the separating different parts of the test with whitespace makes it much more readable
```suggestion
               String expectedAccept = "application/json";
        Request request = Request.builder().build();

        when(deserializer.deserialize(eq(response))).thenReturn("value");
        when(deserializer.accepts()).thenReturn(Optional.of(expectedAccept));
        when(channel.execute(eq(endpoint), any())).thenReturn(Futures.immediateFuture(response));

        ListenableFuture<String> result = DefaultClients.INSTANCE.call(channel, endpoint, request, deserializer);

        assertThat(result).isDone();
        assertThat(result.get()).isEqualTo("value");
        verify(channel).execute(eq(endpoint), requestCaptor.capture());
        assertThat(requestCaptor.getValue().headerParams().get(HttpHeaders.ACCEPT))
                .containsExactly(expectedAccept);
```

Reply:
:+1: good call